sites/gent/macros.tex

```diff
@@ -16,7 +16,28 @@
 % verify with the keys actually on the host with
 % awk '{print $2}' ssh_host_rsa_key.pub | base64 -d | sha256sum -b | awk '{print $1}' | xxd -r -p | base64
 % ssh-keygen -l -f key.pub
-\macro{\loginfingerprint}{RSA: 2f:0c:f7:76:87:57:f7:5d:2d:7b:d1:a1:e1:86:19:f3 (MD5) SHA256:k+eqH4D4mTpJTeeskpACyouIWf+60sv1JByxODjvEKE, ECDSA: 13:f0:11:d1:94:cb:ca:e5:ca:82:21:62:ab:9f:3f:c2 (MD5) SHA256:1MNKFTfl1T9sm6tTWAo4sn7zyEfiWFLKbk/mlT+7S5s, , ED25519: fa:23:ab:1f:f0:65:f3:0d:d3:33:ce:7a:f8:f4:fc:2a (MD5), SHA256:5hnjlJLolblqkKCmRduiWA21DsxJcSlpVoww0GLlagc}
+
+\newcommand{\opensshFirstConnect}{RSA key fingerprint is 2f:0c:f7:76:87:57:f7:5d:2d:7b:d1:a1:e1:86:19:f3
```

Comment:
Maybe include a comment on how you collected this, in case we need to update it at some point (or to let other sites know how to add the missing info)?

Reply:
It's right above in a comment.

Reply:
Ugh, sorry. :)